### PR TITLE
refs can point to an image manifest list

### PR DESCRIPTION
**Pull request by coolljt0725**
Signed-off-by: Lei Jitang <leijitang@huawei.com>

### CODE REVIEW

**Comment by wking** [comment]
0f1d0aff looks good to me.

While we're touching this, maybe make “image manifest” and “image
manifest list” links to manifest.md#image-manifest and
manifest.md#manifest-list respectively?

**Comment by coolljt0725** [comment]
@wking  updated

**Comment by wking** [comment]
17f67c5eb looks good to me.

**Comment by vbatts** [comment]
LGTM

File: image-layout.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Given an image layout a tool can convert a given ref into a runnable OCI Image F
 The image layout has two top level directories:
 
 - "blobs" contains content-addressable blobs. A blob has no schema and should be considered opaque.
-- "refs" contains [descriptors][descriptors]. Commonly pointing to an image manifest.
+- "refs" contains [descriptors][descriptors]. Commonly pointing to an [image manifest](manifest.md#image-manifest) or [an image manifest list](manifest.md#manifest-list).
```
Comment (jonboulle) [inline image-layout.md]: nit, the square bracket should be after the article
`an [image manifest list`

(as it is for image manifest)

**Comment by coolljt0725** [comment]
@jonboulle  updated

**Comment by jonboulle** [comment]
lgtm

**Comment by vbatts** [comment]
LGTM

[Approved with PullApprove](https://pullapprove.com/opencontainers/image-spec/pull-request/335/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=opencontainers/image-spec)

**Pull request merged**